### PR TITLE
Reorganization of yona-log-parser

### PR DESCRIPTION
**Pull request by kti-richard**
Slight refactor and reorganization of the yona-log-parser smoothwall component to better integrate with Smoothwall to allow for easier install, and update/upgrade survivability.  Based on upstream recommendations from Kliksafe.

This setup is active on beta currently.

### CODE REVIEW

File: scripts/smoothwall/HandleDansGuardianLog.pl [hunk 1]
```diff
@@ -1,201 +0,0 @@
-#!/usr/bin/perl
```
Comment (Bert-R) [inline scripts/smoothwall/HandleDansGuardianLog.pl]: Just a quick one: why is this file being deleted?

Comment (kti-richard) [reply scripts/smoothwall/HandleDansGuardianLog.pl]: Moved to modules/yona/bin
(basically, laying it out into into a Smoothwall format)

Comment (kti-richard) [reply scripts/smoothwall/HandleDansGuardianLog.pl]: Git did not pick it up as a move.  I didn't bother trying to figure out why.

Comment (Bert-R) [reply scripts/smoothwall/HandleDansGuardianLog.pl]: I guess you still need to bother, as the "new" Perl script is not part of this PR, so now it comes as a delete, while you were expecting a delete/create and where I hoping for a move, so we retain the history.

Comment (kti-richard) [reply scripts/smoothwall/HandleDansGuardianLog.pl]: Files were missing due to the project gitignore for general 'bin' exclude.

**Comment by kti-richard** [comment]
I am going to close this pull request and submit a new one

**Pull request closed without merging**